fetcher.py

```diff
@@ -10,7 +10,17 @@ def version(self):
     return parse_version(json.loads(self.open('https://mtgjson.com/json/version.json')))
 
   def all_cards(self):
-    return json.loads(self.open('https://mtgjson.com/json/AllCards.json'))
+    if os.path.isdir('./ziptemp'):
+      shutil.rmtree('./ziptemp')
+      
+    os.mkdir('./ziptemp')
+    urllib.request.urlretrieve('https://mtgjson.com/json/AllCards.json.zip', './ziptemp/AllCards.json.zip')
```

Comment:
We should really make "download {url} to {location}" a method of fetcher and call it when fetching images from bluebones or gatherer, too.